go-controller/pkg/kubevirt/dhcp.go

```diff
@@ -0,0 +1,145 @@
+package kubevirt
+
+import (
+	"fmt"
+	"net"
+
+	corev1 "k8s.io/api/core/v1"
+	ktypes "k8s.io/apimachinery/pkg/types"
+	utilnet "k8s.io/utils/net"
+
+	libovsdbclient "github.com/ovn-org/libovsdb/client"
+
+	kubevirtv1 "kubevirt.io/api/core/v1"
+
+	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/config"
+	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/factory"
+	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/libovsdbops"
+	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/nbdb"
+	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/util"
+)
+
+const (
+	dhcpLeaseTime = 3500
+)
+
+type dhcpConfigs struct {
+	V4 *nbdb.DHCPOptions
+	V6 *nbdb.DHCPOptions
+}
+
+func EnsureDHCPOptionsForVM(controllerName string, nbClient libovsdbclient.Client, watchFactory *factory.WatchFactory, pod *corev1.Pod, ovnPodAnnotation *util.PodAnnotation, lsp *nbdb.LogicalSwitchPort) error {
+	// Fake router to delegate on proxy arp mechanism
```

Comment:
This comment just confuses me. What does this mean ? What does the router (which router ?) have to do with the vm name below ?

Reply:
Removing, is a left over from refactorings.